tools/aws_lab_setup/roles/control_node/tasks/main.yml

```diff
@@ -32,31 +32,161 @@
     - control
   when: control_node_install_ansible
 
+- name: Ansible Essentials mode
+  block:
+    - name: Clone lightbulb
+      git:
+        accept_hostkey: yes
+        clone: yes
+        dest: ~{{ username }}/lightbulb
+        repo: https://github.com/ansible/lightbulb.git
+        force: yes
+      become_user: "{{ username }}"
+      tags:
+        - control_node
+        - control
 
-- name: Clone lightbulb
-  git:
-    accept_hostkey: yes
-    clone: yes
-    dest: ~{{ username }}/lightbulb
-    repo: https://github.com/ansible/lightbulb.git
-    force: yes
-  become_user: "{{ username }}"
-  tags:
-    - control_node
-    - control
+    - name: Remove things that students don't need
+      file:
+        state: absent
+        path: ~{{ username }}/lightbulb/{{ item }}
+      with_items:
+        - aws_lab_setup
+        - resources
+        - Vagrantfile
+        - README.md
+      tags:
+        - control_node
+        - control
 
-- name: Remove things that students don't need
-  file:
-    state: absent
-    path: ~{{ username }}/lightbulb/{{ item }}
-  with_items:
-    - aws_lab_setup
-    - resources
-    - Vagrantfile
-    - README.md
-  tags:
-    - control_node
-    - control
+    - name: Create lab inventory directory
+      file:
+        state: directory
+        path: /home/{{ username }}/lightbulb/lessons/lab_inventory
+      tags:
+        - control_node
+        - control
+
+    - name: Put student inventory in proper spot
+      copy:
+        src: ./{{ username }}-instances.txt
+        dest: "{{ control_node_inventory_path }}"
+        owner: "{{ username }}"
+        group: "{{ username }}"
+      when: username in inventory_hostname
+      tags:
+        - control_node
+        - control
+  when: networking is undefined or not networking
+
+- name: Ansible NETWORKING MODE
+  block:
+    - name: Clear out lightbulb directory spot (NETWORKING MODE)
+      file:
+        state: absent
+        path: /tmp/lightbulb
+
+    - name: Clone lightbulb repo (NETWORKING MODE)
+      git:
+        accept_hostkey: yes
+        clone: yes
+        dest: /tmp/lightbulb
+        repo: https://github.com/network-automation/lightbulb.git
```

Comment:
Need to change this to merge.

Reply:
I can't change that until @liquidat 's part gets merged.  That is copying over the instructions, etc that are not part of this PR

Reply:
Can you specify which PR you are talking about? The essentials guide (#250) has been merged. I need to review the Tower one (#262) today.

Reply:
@liquidat hadn't gotten to that guide yet apparently (just talked to him).  I hadn't realize they were all broken apart (makes sense, just didn't realize).  Can we prioritize networking over tower since this PR is in?